NSS-Day-Cohort-53/kennels-voyagers
# can now select an employee when registering an animal

```text
#### Changes Made
1. Modified file `AnimalForm.js` to include fix the ability to choose an employee when registering an animal. Registering the animal now assigns the current user as an owner for that animal, and the employee chosen as the caretaker for that animal
2. Added file `AnimalCaretakerRepository.js` to include related fetch calls for use above
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout src-registerAnimal
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > start the app
    > log in as an owner, not an employee
    > go to animals tab and click on "Register Animal"
    > register an animal and make sure that you can choose an employee from a list of all employees
    > click submit and see if new animal appears in the list of animals
    > check database to see if corresponding animalOwner obj and animalCaretaker obj have been added
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
```
